Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,6 +7,8 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World!");
```

Comment:
This is not what the solution of homework 2 looks like. Please take a look at the README.md file of that branch or read the homework description in Discord again! :)

Reply:
Better now..? *cross my fingers, toes, arms, head, body... everything*

Reply:
Hello world is not needed. Please do take a look at the readme and ask us questions if it's still not clear :)